react/components/InputSearch/index.js

```diff
@@ -94,7 +93,10 @@ class InputSearch extends Component {
                   InputSearch.separatorHeight.regular,
               }}
             />
-            <span className="pointer pl4 c-link" onClick={this.handleSubmit}>
+            <span
+              className="pointer pl4 c-link"
+              onClick={this.handleSubmit}
+              style={{ height: iconSize }}>
```

Comment:
In `small` variation, this is not true. We need a little optical adjustment. Try using `height: 16px` instead of 14px in `small` variation and you'll see the difference.

Reply:
hmm, I didn't understand what you meant, the icons are now fully centralized. 

![image](https://user-images.githubusercontent.com/44899195/91443678-078b1200-e84a-11ea-9475-09bc234e98c7.png)

Reply:
Shouldn't `flex items-center justify-center` classes solve this problem instead of `height` prop?

Reply:
There's a difference between **math alignment** (when you get things centered exactly by their half) and **optical alignment**. In the last case, we adjust things because our eyes make wrong assumptions and think the objects aren't aligned (even if it is mathematically aligned).
In the 🔍  case, the little line below "pulls" everything down. We need to insert a 2px `padding-bottom` to "push up" and align optically.

With span height 14px and svg height 14px:
![Screen Shot 2020-08-27 at 10 28 32](https://user-images.githubusercontent.com/7189694/91448587-6e132e80-e850-11ea-8fe6-582d231f869b.png)

With span height 16px and svg height 14px:
![Screen Shot 2020-08-27 at 10 28 39](https://user-images.githubusercontent.com/7189694/91448598-72d7e280-e850-11ea-87a0-e79e456d779b.png)

Reply:
ahh got it, you should have said that in the bug issue. but ok